<?xml version="1.0" encoding="UTF-8"?>

<launch>


  
  <include file="$(find lynxbot_bringup)/launch/navigation/mapping.launch"/>






  <node name="ydlidar_node"  pkg="ydlidar"  type="ydlidar_node" output="screen" respawn="false" >
    <rosparam file="$(find lynxbot_bringup)/config/start/ydlidar_params.yaml" command="load" />
  </node>



  <node pkg="lynxbot_bringup" type="diff_tf4.py" name="diff_tf" output="screen">
    <rosparam file="$(find lynxbot_bringup)/config/start/odom_encoder_params.yaml" command="load" />
  </node>

  <!--node pkg="rf2o_laser_odometry" type="rf2o_laser_odometry_node" name="rf2o_laser_odometry" output="screen">
    <rosparam file="$(find lynxbot_bringup)/config/rf2o_laser_odometry_params.yaml" command="load" />
  </node-->

  <!--node pkg="robot_localization" type="ekf_localization_node" name="ekf_localization">
    <rosparam file="$(find lynxbot_bringup)/config/ekf_localization.yaml" command="load" />
    <remap from="odometry/filtered" to="odom"/>
  </node-->


  <node name="serial_node" pkg="rosserial_python" type="serial_node.py" output="screen">
      <rosparam file="$(find lynxbot_bringup)/config/start/serial_params.yaml" command="load" />
  </node>
  
</launch>
